tiers/checks.go

```diff
@@ -73,28 +81,38 @@ func getDefaultChecks(nsType string) []innerObjectCheck {
 }
 
 func (a *basicTierChecks) GetExpectedRevisions(awaitility *wait.Awaitility) Revisions {
-	revisions := GetRevisions(awaitility, "basic", "code", "dev", "stage")
-	return revisions
+	return GetRevisions(awaitility.Host(), "basic")
```

Comment:
Since you refactored the `GetRevisions` function, then we don't have any place where the expected set of namespace types would be verified. In other words, with this change we don't verify that for `basic`/`advanced` tiers there should be `dev`, `stage` and `code` ns types and for `team` tier only `dev` and `stage` types (not less not more)

Could you please add such a check somewhere? For example here in the `GetExpectedRevisions` method or into the `nstemplatetier_test.go` file

Reply:
Good point @MatousJobanek. I would probably add such checks to `nstemplatetier_test.go`

Reply:
added assertion to checks in 53bab50